posthog/api/test/test_user.py

```diff
@@ -1,35 +1,92 @@
 from unittest.mock import patch
 
-from posthog.models import Team, User
+from rest_framework import status
 
-from .base import BaseTest
+from posthog.models import Team, User
+from posthog.models.organization import OrganizationMembership
+from posthog.test.base import APITransactionBaseTest, BaseTest
 
 
 class TestUser(BaseTest):
-    TESTS_API = True
-
-    def test_redirect_to_site(self):
-        self.team.app_urls = ["http://somewebsite.com"]
-        self.team.save()
-        response = self.client.get("/api/user/redirect_to_site/?actionId=1")
-        self.assertIn("http://somewebsite.com", response.url)
+    TESTS_API = True  # TODO: Remove once endpoint is refactored to DRF
 
     def test_create_user_with_distinct_id(self):
         with self.settings(TEST=False):
             user = User.objects.create_user(first_name="Tim", email="tim@gmail.com", password=None)
         self.assertNotEqual(user.distinct_id, "")
         self.assertNotEqual(user.distinct_id, None)
 
+    def test_analytics_metadata(self):
```

Comment:
Q: Shouldn't this be under model tests? It's directly testing the model, not API.